packages/amplication-client/src/Project/ProjectPage.scss

```diff
@@ -4,4 +4,21 @@
   display: flex;
   height: 100%;
   flex: 1 auto;
+
+  .commit-page-empty-state {
```

Comment:
why does this class is defined in project.scss?

Reply:
Because that's how we built it when. The page content is of the commits page rendered on the project page. When there are no commits, we render this state on the project page instead the page content.

Reply:
But I see the code that adds the element with the class .commit-page-empty-state in CommitPage

<img width="732" alt="image" src="https://user-images.githubusercontent.com/43705455/189478210-f9f950ca-1f4f-4e60-a543-817c8f4b72cb.png">